### PR TITLE
scss configuration

### PR DESCRIPTION
**Pull request by susird**
This is the first pr for this challenge, titles styles added
image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+package-lock.json
+node_modules
+### OSX ###
+.DS_Store
+.AppleDouble
+.LSOverride
+### Code ###
+# Visual Studio Code - https://code.visualstudio.com/
+.settings/
+.vscode/
```
Comment (josepplloo) [inline .gitignore]: 👀

File: scss/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,2 @@
+$first-font:  'PT Serif', serif;
+$second-font: 'Poppins', sans-serif;
```
Comment (josepplloo) [inline scss/_variables.scss]: new line at the end of the file

File: scss/main.scss [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  padding: 71px 20px 0 20px;
+  display: flex;
+  flex-direction: column
```
Comment (josepplloo) [inline scss/main.scss]: put a semicolon at the end of the line and, order alphabetically

File: index.html [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
```
Comment (josepplloo) [inline index.html]: what happens if you delete this line?

File: index.html [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  <link href="https://fonts.googleapis.com/css2?family=PT+Serif&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=PT+Serif&family=Poppins:wght@300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="css/main.css">
+  <title>My team page</title>
+</head>
+<body>
+  <main class="container">
+    <h1 class="creative-crew-title">The creative crew </h1>
+    <h3 class="we-are-title">Who we are</h3>
+    <p class="information-team">We are team of creatively diverse. driven. innovative individuals working in various locations from the world.</p>
```
Comment (josepplloo) [inline index.html]: If follow the pattern this class needs to be called `team-information`  or `team-information-text`  ?

File: scss/_variables.scss (hunk 2 above)
Comment (susird) [reply scss/_variables.scss]: 8b87577

Comment (susird) [reply scss/_variables.scss]: 8b87577

File: .gitignore (hunk 1 above)
Comment (susird) [reply .gitignore]: What happens with this? :eyes:

File: scss/main.scss (hunk 3 above)
Comment (susird) [reply scss/main.scss]: e2be82f

File: index.html (hunk 5 above)
Comment (anyruizd) [inline index.html]: This shouldn't be here. This PR is supposed to be only a `Hello World`, so please delete it.

File: scss/_variables.scss (hunk 2 above)
Comment (anyruizd) [inline scss/_variables.scss]: Those variables' names don't say anything.

File: scss/main.scss [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  text-transform: uppercase;
+  width: 117px;
+}
+
+.information-team {
+  font-family: $second-font;
+  font-size: 14px;
+  height: 87px;
+  width: 333px;
+}
```
Comment (anyruizd) [inline scss/main.scss]: This shouldn't be here since this PR is just a `Hello World`.

File: .gitignore (hunk 1 above)
Comment (anyruizd) [reply .gitignore]: image

**Review by anyruizd — CHANGES_REQUESTED** [review]

File: scss/main.scss (hunk 6 above)
Comment (susird) [reply scss/main.scss]: 551c90c

File: index.html (hunk 5 above)
Comment (susird) [reply index.html]: 551c90c

**Review by anyruizd — APPROVED** [review]
👍🏽 next time update the readme to reflect the changes

**Pull request merged**